From af7d8a2320463c7ec6ed21f3e96f571a931eb1bb Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Fri, 2 Aug 2024 21:11:01 +0530
Subject: [PATCH] Updated media c2 hal from hidl to aidl.

Updated android.hardware.media.c2 hal from hidl V1.0 to aidl V1.
updated sepolicy to run media c2 aidl.

Tests: Prepared EB and its booted successfully.

Tracked-On: OAM-122196
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 codec2/msdk-codec2/file_contexts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/codec2/msdk-codec2/file_contexts b/codec2/msdk-codec2/file_contexts
index 5963a5c..7db1f35 100644
--- a/codec2/msdk-codec2/file_contexts
+++ b/codec2/msdk-codec2/file_contexts
@@ -1 +1 @@
-/vendor/bin/hw/hardware\.intel\.media\.c2@1\.0-service u:object_r:codec2_hal_default_exec:s0
+/vendor/bin/hw/android\.hardware\.media\.c2-service\.intel u:object_r:codec2_hal_default_exec:s0
-- 
2.34.1

